raw_description/urdf/base_long/base.urdf.xacro

```diff
@@ -64,6 +64,15 @@
     <xacro:batteries_gazebo name="${name}" />
   </xacro:macro>
 
+  <xacro:macro name="box_inertia" params="width length height mass *origin">
+    <inertial>
+      <mass value="${mass}" />
+      <xacro:insert_block name="origin" />
+      <inertia ixx="${0.0833333 * mass * (width * width + height * height)}" ixy="0.0" ixz="0.0"
+        iyy="${0.0833333 * mass * (height * height + length * length)}" iyz="0.0"
+        izz="${0.0833333 * mass * (width * width + length * length)}" />
+    </inertial>
+  </xacro:macro>
```

Comment:
no need to define the macro again....use it from [cob_description/common.xacro](https://github.com/ipa320/cob_common/blob/indigo_dev/cob_description/urdf/common.xacro#L46-L54) as that file already is included....